dfastmi/io/ConfigFileOperations.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+"""
```

Comment:
should this header be added to every file?

Reply:
This is automated, should be in all IP files. @MRVermeulenDeltares create issue. Maarten Klapwijk may know more about this automated.

Reply:
https://issuetracker.deltares.nl/browse/DFAST-116